go.mod

```diff
@@ -28,6 +28,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require golang.org/x/text v0.3.7
```

Comment:
Can we add this dependency declaration to the above `require` section?

Reply:
done